definitions/output/all/reprocess_pages.js

```diff
@@ -0,0 +1,266 @@
+    '$._metadata',
+    '$._detected',
+    '$._detected_apps',
+    '$._detected_technologies',
+    '$._detected_raw',
+    '$._custom',
+    '$._00_reset',
+    '$._a11y',
+    '$._ads',
+    '$._almanac',
+    '$._aurora',
+    '$._avg_dom_depth',
+    '$._cms',
+    '$._Colordepth',
+    '$._cookies',
+    '$._crawl_links',
+    '$._css-variables',
+    '$._css',
+    '$._doctype',
+    '$._document_height',
+    '$._document_width',
+    '$._Dpi',
+    '$._ecommerce',
+    '$._element_count',
+    '$._event-names',
+    '$._fugu-apis',
+    '$._generated-content',
+    '$._has_shadow_root',
+    '$._Images',
+    '$._img-loading-attr',
+    '$._initiators',
+    '$._inline_style_bytes',
+    '$._javascript',
+    '$._lib-detector-version',
+    '$._localstorage_size',
+    '$._markup',
+    '$._media',
+    '$._meta_viewport',
+    '$._num_iframes',
+    '$._num_scripts_async',
+    '$._num_scripts_sync',
+    '$._num_scripts',
+    '$._observers',
+    '$._origin-trials',
+    '$._parsed_css',
+    '$._performance',
+    '$._privacy-sandbox',
+    '$._privacy',
+    '$._pwa',
+    '$._quirks_mode',
+    '$._Resolution',
+    '$._responsive_images',
+    '$._robots_meta',
+    '$._robots_txt',
+    '$._sass',
+    '$._security',
+    '$._sessionstorage_size',
+    '$._structured-data',
+    '$._third-parties',
+    '$._usertiming',
+    '$._valid-head',
+    '$._well-known',
+    '$._wpt_bodies',
+    '$._blinkFeatureFirstUsed',
+    '$._CrUX'
+  ) AS payload,
+  JSON_REMOVE(
+    SAFE.PARSE_JSON(summary, wide_number_mode => 'round'), 
+    '$._adult_site',
+    '$.archive',
+    '$.avg_dom_depth',
+    '$.crawlid',
+    '$.createDate',
+    '$.doctype',
+    '$.document_height',
+    '$.document_width',
+    '$.label',
+    '$.localstorage_size',
+    '$.meta_viewport',
+    '$.metadata',
+    '$.num_iframes',
+    '$.num_scripts_async',
+    '$.num_scripts_sync',
+    '$.num_scripts',
+    '$.pageid',
+    '$.PageSpeed',
+    '$.rank',
+    '$.sessionstorage_size',
+    '$.startedDateTime',
+    '$.url',
+    '$.urlhash',
+    '$.urlShort',
+    '$.usertiming',
+    '$.wptid',
+    '$.wptrun'
+  ) AS summary,
+  STRUCT<
+    a11y JSON,
+    cms JSON,
+    cookies JSON,
+    crux JSON,
+    css_variables JSON,
+    ecommerce JSON,
+    element_count JSON,
+    javascript JSON,
+    markup JSON,
+    media JSON,
+    origin_trials JSON,
+    performance JSON,
+    privacy JSON,
+    responsive_images JSON,
+    robots_txt JSON,
+    security JSON,
+    structured_data JSON,
+    third_parties JSON,
+    well_known JSON,
+    wpt_bodies JSON,
+    other JSON
+  >(
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.a11y"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.cms"),
+    JSON_QUERY(SAFE.PARSE_JSON(custom_metrics, wide_number_mode => 'round'), "$.cookies"),
+    JSON_QUERY(SAFE.PARSE_JSON(payload, wide_number_mode => 'round'), "$._CrUX"),
```

Comment:
I thought we were going to put this in `summary` table not `custom_metrics`?

Reply:
Oh, I followed a "separate column" hint.

Are we cool doubling the size of `summary`?
I believe it's 500Gb vs 1Tb per month in the reprocessed table.

Reply:
But you only pay for the JSON objects you reference, right?

Reply:
No, querying any of these will cost the same:
- summary
- summary._CrUX
- summary.bytesCss

Reply:
Seems it was a wrong estimate, so more like 50Gb all together, not that costly..
So you can check the reprocessed table now with CrUX in `summary`.